fixtures/fields/ArrayField/with-uiSchema.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+  schema: {
+    type: "array",
+    title: "title",
+    items: {
+      type: "string",
+      title: "item"
+    }
+  },
+  uiSchema: {
+    items: {
+      widget: "textarea"
+    }
+  },
+  formData: ["item1", "item2"],
+  onChange: console.log.bind(console)
```

Comment:
Do you want to `console.log.bind(console, "change")` like you did above for `submit` and `errors`?